internal/store/project_ip_access_lists.go

```diff
@@ -17,36 +17,38 @@ package store
 import (
 	"errors"
 	"fmt"
+	"time"
 
 	"github.com/mongodb/mongodb-atlas-cli/internal/config"
 	atlas "go.mongodb.org/atlas/mongodbatlas"
+	atlasv2 "go.mongodb.org/atlas/mongodbatlasv2"
 )
 
 //go:generate mockgen -destination=../mocks/mock_project_ip_access_lists.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/store ProjectIPAccessListDescriber,ProjectIPAccessListLister,ProjectIPAccessListCreator,ProjectIPAccessListDeleter
 
 type ProjectIPAccessListDescriber interface {
-	IPAccessList(string, string) (*atlas.ProjectIPAccessList, error)
+	IPAccessList(string, string) (*atlasv2.NetworkPermissionEntry, error)
 }
 type ProjectIPAccessListLister interface {
-	ProjectIPAccessLists(string, *atlas.ListOptions) (*atlas.ProjectIPAccessLists, error)
+	ProjectIPAccessLists(string, *atlas.ListOptions) (*atlasv2.PaginatedNetworkAccess, error)
 }
 
 type ProjectIPAccessListCreator interface {
-	CreateProjectIPAccessList([]*atlas.ProjectIPAccessList) (*atlas.ProjectIPAccessLists, error)
+	CreateProjectIPAccessList([]*atlas.ProjectIPAccessList) (*atlasv2.PaginatedNetworkAccess, error)
 }
 
 type ProjectIPAccessListDeleter interface {
 	DeleteProjectIPAccessList(string, string) error
```

Comment:
Delete e2e tests are still failing due to versioning error:
`{"detail":"Invalid accept header or version date.","error":406,"errorCode":"INVALID_VERSION_DATE","parameters":[],"reason":"Not Acceptable"}
Error: 406 Not Acceptable
`

Reply:
Working on fix.

Reply:
no rush :)